modules/zarr/src/index.ts

```diff
@@ -1,3 +1 @@
-export { loadBioformatsZarr, loadOmeZarr } from './zarr';
-
-export { default as ZarrPixelSource } from './zarr/pixel-source';
+export { load } from './lib';
```

Comment:
Nit: I'd suggest [having newlines](https://stackoverflow.com/q/729692) at the end of all files. IMO this is something loaders.gl CI (or probably ocular-dev-tools under the hood) should enforce.